app/cloud_tasks/cloud_task_publishers.py

```diff
@@ -9,11 +10,14 @@
 
 
 class CloudTaskPublisher:
-    def __init__(self):
+    def __init__(self, queue_name: str):
         self._client = CloudTasksClient()
-
         _, self._project_id = auth.default()
 
+        self._parent = self._client.queue_path(
+            self._project_id, "europe-west2", queue_name
+        )
```

Comment:
Yep my bad. As we are only publishing to a single queue it was more efficient to do this work once. I'll move it back so that we have the option of publishing to multiple queues